### ISSUE TITLE
Terms & Agreements checkbox cannot be used fully with VoiceOver

### ISSUE THREAD

**Issue by charlag** [tutao/tutanota#8292]
Email address selection page of the signup wizard has two checkboxes. One of them is "I have read and agree to the following documents: General terms and conditions, Privacy policy". The latter two are separate links but they are still contained within the checkbox element.

With VoiceOver it's not possible to focus on the links.

When selecting the checkbox some users report that the link is opened instead:
https://github.com/tutao/tutanota/issues/8287

In our own testing the checkbox is ticked but the link cannot be opened.

Neither is acceptable. It should be possible to focus and trigger the links separately from the checkbox.

**Test Notes**
- [ ] Signup through iOS with voice-over enabled. Navigate to Terms and Conditions page.
- [ ] Able to reach "General terms and conditions & Privacy policy" links individually, Independent from "I have read and agree"
